tests/functional/test_tfrecord.py

```diff
@@ -16,7 +16,7 @@
 import os
 from click.testing import CliRunner
 from glob import glob
-from imagine import main
+from imagine.imagine import main
```

Comment:
For this and the other tests, will users need to import main from imagine.imagine, or will it look more like the test_units.py code where one can import in a less repetitious way?

Reply:
This update just adds the ability for users to be able to install the package and run it from the CLI doing something like

```bash
imagine create-images ...
```

While users _can_ import the library into their code with this update, I think we need to make some larger updates to the code and add more formal documentation to get it the way we would expect other users to be able to import. I am planning on working on that for the next update, but this doesn't support that functionality at the moment. If you think it belongs here now, I'd be happy to include it!

Reply:
Understood - nah, not needed yet, just wanted to understand how we wanted the library aspect to work. If that's a future feature and this is just test kludge for now, I can live with that. :)

Reply:
I will add that to the roadmap!